src/javascript/static/pages/static_pages.js

```diff
@@ -46,4 +46,7 @@ module.exports = {
         onLoad  : () => { MenuSelector.init(['general', 'account-management', 'referral-tools']); },
         onUnload: () => { MenuSelector.clean(); },
     },
+    BinaryInNumbers: {
+        onLoad: () => { Scroll.scrollToHashSection(); },
```

Comment:
the first time that this page loads, e.g. in a new incognito tab instance, it is not able to scroll to the correct section. perhaps the images load after the scrolling is done and pushes the page down due to that. I suggest adding a small timeout to overcome that. if you have a better solution, that's fine too.